ciao_contrib/_tools/specextract.py

```diff
@@ -109,6 +110,14 @@ def __init__(self,params):
         self.wmap_clip = params["wmap_clip"]
         self.wmap_threshold = params["wmap_threshold"]
 
+        if params["parallel"].lower() == "no":
```

Comment:
I note that this logic is also in `bin/specextract` - I forget with the split up whether this is expected or not?

Reply:
it is, there's some lightweight parallelization going on in here.  If we're alright with just forcing the parallelization and not trying to control it, I'd happily remove this logic.

Reply:
Let's leave that for a later cleanup.